pkg/metering/v1/logs.go

```diff
@@ -19,8 +20,10 @@ func NewLogs(logger *zap.Logger) metering.Logs {
 }
 
 func (meter *logs) Size(ld plog.Logs) int {
-	total := 0
 
+	meter.Logger.Debug("Calculating logs size")
```

Comment:
We can remove this @shivanshuraj1333. Most of our services run in debug mode. It will hog the logs.